### ISSUE TITLE
Update meeting - buttons not working

### ISSUE THREAD

**Issue by mewa1024** [microsoft/botframework-components#409]
## Describe the bug
Updating a meeting via typing didn't quite work (the skill didn't seem to understand my update), and updating via the "Update event" button the event card caused the skill to show the card again, which felt like going in a loop.

## To Reproduce
1. Ask the skill to show an event:
2. Press the "Update event" button. The skill asks what you'd like to update but the repeats the card.
3. Type to the skill to change the event time.
4. The skill suggested two times, but then did not understand when I selected one of the suggested actions.

Here is a screenshot of the interaction:

![image](https://user-images.githubusercontent.com/39317542/99445014-d6aefc00-28d1-11eb-97ae-0ef045980fe1.png)

## Expected behavior
After the bot asks "Tell me what you'd like to update", I expected to see either suggested actions or a card that would show me options of things I could update. Instead it repeated the card, which feels a little loopy:
![image](https://user-images.githubusercontent.com/39317542/99445163-1675e380-28d2-11eb-9569-5951f1de5d09.png)

I also expected the suggested time change to work, since it was shown as a suggested action.
![image](https://user-images.githubusercontent.com/39317542/99445321-402f0a80-28d2-11eb-9aa3-29373cc26bd2.png)
(also, I wasn't sure what the "20" referred to.)

Minor: the wording of "Can you ask me differently?" is a little awkward. Maybe "Can you ask me in a different way?" or "Can you rephrase your request?"

**Comment by xhr0804** [comment]
Looks like there are there issues here:
1. Card repeat, @KayMKM could you please check it
2. Time prompt choice
3. Error message

**Comment by Quirinevwm** [comment]
Agree to take the loop card out, so only ask in transcript what the user wants to update - take out the 20 (bug) in duration - can we check why it was not possible to complete? @KayMKM 
- Need to update training for duration by adding "end time to" @lauren-mills 

I've update the transcript file for 'Can you ask me differently?" into:
- I'm not sure I understand. Can you please try again?
- Oops, looks like I'm stuck. **Can you try to ask me in a different way?**
- Looks like I'm all mixed up. Let's try asking again, but maybe rephrase your request?

**Comment by KayMKM** [comment]
@Quirinevwm Sure the duplicate card issue should be fixed in #418, and other issues will be fixed in #440
